access-control-app/project.clj

```diff
@@ -105,9 +105,17 @@
             ;; Kaocha test aliases
             ;; refer to tests.edn for test configuration
             "kaocha" ["with-profile" "+kaocha" "run" "-m" "kaocha.runner"]
-            "itest" ["kaocha" "--focus" "integeration"]
-            "utest" ["kaocha" "--focus" "unit"]
-            "ci-test" ["kaocha" "--profile" ":ci"]
+            "itest" ["kaocha" "--focus" ":integration"]
```

Comment:
Add comments on what these targets are for and explain why the ci build is different from local.